app/Models/Model.php

```diff
@@ -19,7 +19,7 @@
 
 abstract class Model extends BaseModel
 {
-    use HasFactory, Traits\FasterAttributes, Validatable;
+    use HasFactory, Traits\FasterAttributes, Traits\IncrementInstance, Validatable;
```

Comment:
not very related but one of the more recent coding styles says to import one trait at a time

> Each individual trait that is imported into a class MUST be included one-per-line and each inclusion MUST have its own `use` import statement.

https://www.php-fig.org/per/coding-style/#42-using-traits